client/src/component/Video.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+
+import DeleteButton from "./DeleteButton";
+import Rating from "./Rating";
+
+function Video({ videoData }) {
+  const [removedId, setRemovedId] = useState([]);
+
+  function handleClick(id) {
```

Comment:
In React we use arrow functions, not function declaration :)

Sorry Laleh, I know I am being annoying 😅  I just want to keep saying it so we remember :)